Mage.Sets/src/mage/cards/e/ExuberantFuseling.java

```diff
@@ -0,0 +1,58 @@
+package mage.cards.e;
+
+import mage.MageInt;
+import mage.abilities.common.EntersBattlefieldTriggeredAbility;
+import mage.abilities.common.PutIntoGraveFromBattlefieldAllTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.CountersSourceCount;
+import mage.abilities.effects.common.continuous.BoostSourceEffect;
+import mage.abilities.effects.common.counter.AddCountersSourceEffect;
+import mage.abilities.keyword.TrampleAbility;
+import mage.abilities.meta.OrTriggeredAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.SubType;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import mage.filter.StaticFilters;
+
+import java.util.UUID;
+
+public class ExuberantFuseling extends CardImpl {
+
+    private static final DynamicValue oilCounters = new CountersSourceCount(CounterType.OIL);
+
+    public ExuberantFuseling(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{R}");
+        this.addSubType(SubType.PHYREXIAN);
+        this.addSubType(SubType.GOBLIN);
+        this.addSubType(SubType.WARRIOR);
+        this.power = new MageInt(0);
+        this.toughness = new MageInt(1);
+
+        //Trample
+        this.addAbility(TrampleAbility.getInstance());
+
+        //Exuberant Fuseling gets +1/+0 for each oil counter on it.
+        this.addAbility(new SimpleStaticAbility(new BoostSourceEffect(oilCounters, oilCounters, Duration.WhileOnBattlefield)));
```

Comment:
This boost effect should only increase power, not toughness, currently incorrectly grants +1/+1 for each oil counter on the permanent

Reply:
done (bad copy and paste ;)